lib/ts/BufferWriter.h

```diff
@@ -24,6 +21,10 @@
     limitations under the License.
  */
 
+#if !defined TS_BUFFERWRITER_H_
+#define TS_BUFFERWRITER_H_
```

Comment:
Why not just switch to **#pragma once** if you're going to futz with this stuff?

Reply:
We've had long discussions on `#pragma` and I'm still not sufficiently confident to move on.